web/css/print.css

```diff
@@ -0,0 +1,49 @@
+/* Common */
```

Comment:
try to put this content on `front/style/print.scss` and write a webpack rule to generate `web/css/print.css` from `front/style/print.scss` :)